gcp/key-vendor/app.yaml

```diff
@@ -0,0 +1,22 @@
+runtime: go111
+
+env_variables:
+  # EVALUATOR is a text-encoded proto builder.Validator.  It configures the
+  # complete parsing and validation pipeline.
+  EVALUATOR: |
+    parser {
+      shims {
+        elixir {
+          client_id: "your-client-id-here"
+        }
+      }
+    }
+    validator {
+      simple {}
```

Comment:
Why not something like constant { true }?  I realize it's the same thing but it's not clear unless you read the source for the simple validator

Reply:
You're right, it's unclear.  I've added 'constant' to the builder proto.  I also toyed with the idea of making an empty Validator proto mean always succeeds, but don't like the idea of failing open.